Update readme for more details on setup (#15)
supercrafter100/Nameless-Suggestions-Discord-Bot: README.md

3. Install all dependencies by executing `npm install -D`
4. Build the bot `npx tsc`
5. Create a `.env` file and put in the contents of the `.example.env` file that is included. Then fill in the values
   - `TOKEN`: This should be your Discord Bot's token</br>
   - `GUILDID`: This should be the ID of your Discord Server [how?](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)</br>

>    - `DOMAIN`: This should be the URL of the SERVER you're hosting the bot on. For example, if your IP is `5.5.5.5`, and you're running the bot on port `4032` with no SSL, your `DOMAIN` would be `http://5.5.5.5:4032`</br>

Reviewer: This doesn't *have* to be the ip of the bot. If you're using a reverse proxy like I am, you can configure a domain for it too! Though I guess generally people probably won't be putting as much effort in doing that.

>    - `DB_*`: These values should be the same as your nameless mc database.</br>

Reviewer: I generally don't advice using the same database as your namelessmc one for this. While it's completely possible, ideally  you should just have a separate one for this bot.

>    - `DB_*`: These values should be the same as your nameless mc database.</br>

Reviewer: I just added an additional env variable in the `.example.env` called PORT. This was already in the code I just never documented it. Might as well add it here as well? It defaults to port 3000 if it's not specified.

7. Run the bot using `npm run start`

## Setup

> 1. Ensure the bot is invited to the server (one you specified in `GUILDID`)

Reviewer: ```suggestion
1. Ensure the bot is invited to the server (the one you specified in the `GUILDID` environmental variable)
```

2. Run `/settings set apikey
   > The options should be the same (or similar if reverse proxied) as the ones in Staff CP -> Configuration -> API

> 3. This should automatically create a webhook in NamelessMC, you can check in Staff CP -> Configuration -> Webhooks

Reviewer: ```suggestion
3. This should automatically create a webhook in NamelessMC, you can check in StaffCP -> Configuration -> Webhooks
```

   > If it didn't, run `/setup` and follow the instructions there
4. Finally, just run `/settings set suggestionchannel`
   > The option should be the name of the channel you want suggestions to pop into. ( This can only be a text channel, nothing else. )

> **NOTE**: If you want users to be able to react and write in suggestion threads. They require to have their discord account linked to their NamelessMC one. This requires you to set up the nameless-link discord bot so users can achieve this.

Reviewer: ```suggestion
**NOTE**: If you want users to be able to react and write in suggestion threads. They require to have their discord account linked to their NamelessMC one. This requires you to set up the [nameless-link discord bot](https://github.com/NamelessMC/Nameless-Link) so users can achieve this.
```
